src/components/Button.tsx

```diff
@@ -54,7 +54,6 @@ class Button extends React.Component<
             danger,
             disabled,
             name,
-            //eslint-disable-next-line @typescript-eslint/camelcase
```

Comment:
Why is this rule not needed anymore?

Reply:
This rule does not exist anymore: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/camelcase.md